### DISCUSSION TITLE
Add second inverter

### DISCUSSION THREAD

**Discussion by lukinfo** [StealthChesnut/HA-FoxESS-Modbus#85] [Q&A]
Hi,
very good job. Could you tell me how to add a second inverter? Is it possible at all? Is it possible to add sum  produced energy ?

**Comment by MadWayne1** [comment]
I dont have the CT2 clamp on my solis 2kw invertor so cant see it in Fox Ess Cloud and dont see the point other than to call it into HA with regular updates, dont think FoxEss - Modbus even looks at CT2 clamp as it stands,

However it would be great if  could call my solis data into this FoxEss - Modbus  as to make more sense of the numbers.

Currently the load  will be sat 200W taken from the Grid CT clamp, however what happens when the old panels as there output increases is the load diminishes to zero and then goes negative as the solar output gets above the back ground load.

If I can get a solis  HA mod  so i can pull in the data first, I would just then need to integrate the data into the Fox Ess  i can see it all in one place and do total sola, or am i over thinking it?

The other issue is the solis is  AC only, and the invertor only becomes accessible once  there is sufficient day light to power up the invertor unlike the fox hybrid invertor which is always on.
